ts/components/staking/wizard/wizard_flow.tsx

```diff
@@ -432,7 +432,7 @@ const MarketMakerStakeInputPane = (props: MarketMakerStakeInputPaneProps) => {
             <PoolsContainer>
                 <MarketMaker
                     key={marketMakerPool.poolId}
-                    name={marketMakerPool.metaData.name || utils.getAddressBeginAndEnd(marketMakerPool.operatorAddress)}
```

Comment:
We were a bit inconsistent using both `poolId` and `operatorAddress` as a fallback if no name exists so I standardized it to use `poolId`.